### PR TITLE
🌱 Uplift BMO and IPAM in go modules and config/

### PR DESCRIPTION
**Pull request by kashifest**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -87,7 +87,7 @@ require (
 	github.com/mailru/easyjson v0.7.7 // indirect
 	github.com/mattn/go-isatty v0.0.16 // indirect
 	github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369 // indirect
-	github.com/metal3-io/baremetal-operator/pkg/hardwareutils v0.1.1 // indirect
+	github.com/metal3-io/baremetal-operator/pkg/hardwareutils v0.1.2 // indirect
```
Comment (furkatgofurov7) [inline go.mod:90]: this is not latest, I am seeing the same locally

Comment (kashifest) [reply go.mod]: Good Catch! fixed

Comment (furkatgofurov7) [reply go.mod]: and it should be coming from https://github.com/metal3-io/baremetal-operator/blob/v0.2.0/apis/go.mod#L7?

Comment (kashifest) [reply go.mod]: Yes you are right :(

**Review by lentzi90 — COMMENTED** [review]
/lgtm

File: go.mod (hunk 1 above)
Comment (kashifest) [reply go.mod]: but as I have uplifted here, it should still be fine, BMO issue has to be solved separately.

**Review by furkatgofurov7 — APPROVED** [review]

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Pull request merged**